src/main/FizzBuzz.java

```diff
@@ -0,0 +1,24 @@
+package main;
+
+public class FizzBuzz {
+    public static void main(String[] args) {
+        for (int i=1; i<=100; i++){
+            if (i%3==0 && i%5==0){
```

Comment:
Please use "if(condition)...else if(contion).." expression because when you get a number, for example, 15, your program will print all your messages, but according to the task your program should pring only one message "FizzBuzz".